### DISCUSSION TITLE
Subscribe to Issue

### DISCUSSION THREAD

**Discussion by fishcharlie** [community/community#8389] [Mobile]
It should be possible to subscribe to email notifications for an issue on the mobile app (similar to the web version).

**Comment by eliperkins** [comment]
Thanks for the feedback, @fishcharlie! To subscribe to an issue (for both GitHub notifications and for email notifications), tap the `•••` (overflow) menu at the top of the issue and choose "Subscribe".

Here's a screenshot of how it looks (the screenshot says "Unsubscribe", but the action will be in the same place!)

<img width="431" alt="Screen Shot 2021-12-02 at 3 12 26 PM" src="https://user-images.githubusercontent.com/1051453/144679612-d8c0e13e-81a3-410d-99d5-d90f50356b51.png">

**Reply by fishcharlie** [reply]
Ahh. Thanks so much.

Just from a UI/UX perspective, I expected it to be in the bottom modal thing along with the close issue, tags, milestone, comments, etc section. Just because that is near where it's located on the web version.

Maybe the placement currently is more native to iOS and better. It's really hard to say.

Might be worth discussing with the team tho to see if the placement is truly in the best place.
